WordPressKit/ActivityServiceRemote.swift

```diff
@@ -37,28 +37,31 @@ open class ActivityServiceRemote: ServiceRemoteWordPressComREST {
                                    group: [String] = [],
                                    success: @escaping (_ activities: [Activity], _ hasMore: Bool) -> Void,
                                    failure: @escaping (Error) -> Void) {
-        let endpoint = "sites/\(siteID)/activity"
-        let path = self.path(forEndpoint: endpoint, withVersion: ._2_0)
-        let pageNumber = (offset / count + 1)
-        var parameters: [String: AnyObject] = [
-            "number": count as AnyObject,
-            "page": pageNumber as AnyObject
-        ]
 
-        if !group.isEmpty {
-            parameters["group[]"] = group.joined(separator: ",") as AnyObject
-        }
+        var path = URLComponents(string: "sites/\(siteID)/activity")
+
+        path?.queryItems = group.map { URLQueryItem(name: "group[]", value: $0) }
+
+        let pageNumber = (offset / count + 1)
+        path?.queryItems?.append(URLQueryItem(name: "number", value: "\(count)"))
+        path?.queryItems?.append(URLQueryItem(name: "page", value: "\(pageNumber)"))
 
         if let after = after, let before = before,
            let lastSecondOfBeforeDay = Calendar.current.date(byAdding: .second, value: 86399, to: before) {
```

Comment:
I know this isn't new, but I just had a thought about moving moving `86399` to an extension since it's used in a couple places? 

`Calendar.current.endOfDay(for date: Date) -> Date?`

Reply:
Done in 1e13a77d9a5d1809425b611ea392e19d87d84e7d